client/state/data-layer/wpcom/sites/activity/from-api.js

```diff
@@ -19,48 +19,44 @@ export const DEFAULT_GRIDICON = 'info-outline';
 /**
  * Transforms API response into array of activities
  *
- * @param  {object} apiResponse                      API response body
- * @param  {array}  apiResponse.current.orderedItems Array of item objects
- * @return {array}                                   Array of proccessed item objects
+ * @param  {Object} apiResponse                      API response body
+ * @param  {Array}  apiResponse.current.orderedItems Array of item objects
+ * @return {Array}                                   Array of proccessed item objects
  */
 export function transformer( apiResponse ) {
 	const orderedItems = get( apiResponse, [ 'current', 'orderedItems' ], [] );
 	return map( orderedItems, processItem );
 }
 
+export function processItemActor( actor ) {
+	return {
+		actorAvatarUrl: get( actor, [ 'icon', 'url' ], DEFAULT_GRAVATAR_URL ),
```

Comment:
Why is this `DEFAULT_GRAVATAR_URL` defined and exported in line 16 above? It's only used in this line. Same applies to `DEFAULT_GRIDICON`.
Is there a reason for doing it and export them? Both are only used once and only in this file. Maybe they should be replaced with its values inline.

Reply:
I'd speculate that the export is leftover from tests of an earlier implementation of `fromApi` that was significantly more complex.

As for defining constants rather than inlining, it made sense when they needed to be exported. At this point they could be inlined, but I think there's some clarity provided by naming them `DEFAULT_…`, but no strong feelings either way.